Bill3.java

```diff
@@ -1,8 +1,23 @@
-// Splits a restaurant bill evenly among three diners.
-public class Bill3 {
-	public static void main(String[] args) {
-		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+
+public class Bill3
+ {
+	public static void main(String[] args)
+    {
+		// input of three names and full price of bill
+		String name1=args[0];
+		String name2=args[1];
+		String name3=args[2];
+
+		int Fullprice = Integer.parseInt(args[3]);
```

Comment:
The variable Fullprice does not follow Java naming conventions. Variable names should start with a lowercase letter and use camelCase - fullPrice = Integer.parseInt(args[3]);